relay-metrics/src/aggregator.rs

```diff
@@ -1571,4 +1588,42 @@ mod tests {
         ]
         "###);
     }
+
+    #[test]
+    fn test_get_flush_time_with_backdated_bucket() {
```

Comment:
I'd prefer if the test would test the aggregator, we should be able to fast forward time with [`tokio::time::advance`](https://docs.rs/tokio/latest/tokio/time/fn.advance.html).

Reply:
Talked about it, doesn't seem feasible. Let's keep the test as is.